packages/connector-jsdom/package.json

```diff
@@ -61,7 +61,7 @@
   },
   "repository": "webhintio/hint",
   "scripts": {
-    "build": "npm run clean && npm-run-all build:*",
+    "build": "npm-run-all build:*",
```

Comment:
IMO, we still needs a build with the `npm run clean` in case we are just testing a package.

Reply:
Maybe we can have `build` as it was, and then a `build-all` without the clean.

Reply:
I'm fine with not automatically running `clean` on every build and leaving it for explicit use. I think running `clean` for a full build from the root is still a good idea, which this PR still does, so I'm good with the changes.

Reply:
OK, can you 👍👎 this PR whenever you have the time and I'll fix the conflicts with `yarn.lock` and get it merged right after.